smoke-tester/src/tests.rs

```diff
@@ -230,7 +230,11 @@ pub fn test_flashing(tracker: &TestTracker, session: &mut Session) -> Result<(),
 
     let start_time = Instant::now();
 
-    download_file_with_options(session, test_binary, Format::Elf, options)
+    let format = match session.target().default_format {
+        probe_rs_target::BinaryFormat::Idf => Format::Idf(Default::default()),
```

Comment:
Nice to cover this too :)

Reply:
It's slightly unfortunate that `FlashOptions` is private to the probe-rs binary and so it can't be used here, that type includes this logic.

Reply:
Hmm that's unfortunate indeed.